src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/ExceptionDeserializationException.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    [Serializable]
+    public class ExceptionDeserializationException : Exception
+    {
+        public ExceptionDeserializationException()
+            : base("This exception could not be deserialized.")
+        {
+        }
```

Comment:
Could this be a more general nuget serialization exception, similar to what JsonException is to JSON.Net? This allows for reuse by any of our custom converters. Maybe NuGetJsonException?

Also, I would support passing a JsonException as an inner exception.

Reply:
I don't think sharing exceptions between converters provides much value. If anything we'd want them to be separate types so we can catch individual issues.

I've actually decided to get rid of this exception type and just return the `Exception` that was originally thrown.